com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/helpers/SystemParameters.java

```diff
@@ -174,8 +174,8 @@ private String appendRoutePathIfPresent(String uri, Map<String, Object> modulePa
     private String getDefaultHost(String moduleName) {
         String host = (targetName + " " + moduleName).replaceAll("\\s", "-")
                                                      .toLowerCase();
-        if (!HOST_VALIDATOR.isValid(host)) {
-            return HOST_VALIDATOR.attemptToCorrect(host);
+        if (!HOST_VALIDATOR.isValid(host, null)) {
```

Comment:
I think it will be better to return previous state of "isValid" and "attemptToCorrect" methods and add as default methods "isValid" and "attemptToCorrect" methods with 2 parameters. In this way you wouldn't need to pass null values to all implementations of ParameterValidator interface.